apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -245,6 +245,8 @@ def fabric_l3(self):
         if self._fabric_l3 is None:
             self._fabric_l3 = self._cisco_l3 or hasattr(self._l3_plugin,
                                                         '_apic_driver')
+            # Update the apicapi's property
+            self.apic_manager.apic_config.enable_aci_routing = self._fabric_l3
```

Comment:
I'm not sure this is the right thing to do. This relies on the proper code in mechanism driver getting called in order to set the config option ... what if apic_manager has some code which runs before that and uses this flag?